src/pages/documentation/faq.md

```diff
@@ -58,7 +58,7 @@ The Experience Platform SDK migrates the locally stored user contexts from the 4
 
 ### What platforms are supported?
 
-For a complete list of supported platforms, please read the [latest SDK versions document](./upgrade-platform-sdks/current-sdk-versions.md).
+For a complete list of supported platforms, please read the [latest SDK versions document](./current-sdk-versions.md).
```

Comment:
I don't think this link will work, because I believe the relative path is based off of where it is within the repository rather than on the site.

(Unless you tested it and it does work, then you can disregard this comment!)

Reply:
Worked when tested on localhost